lua/midi.lua

```diff
@@ -186,13 +186,24 @@ function Midi.to_msg(data)
   local msg = {}
   -- note on
   if data[1] & 0xf0 == 0x90 then
-    --print("note")
-    msg = {
-      type = "note_on",
-      note = data[2],
-      vel = data[3],
-      ch = data[1] - 0x90 + 1
-    }
+    if data[3] > 0 then
+      --print("note")
```

Comment:
no need for these commented out lines, the comment about this velocity check could be moved before this block.